tests/conftest.py

```diff
@@ -65,3 +74,60 @@ def image(dev_image):
 def store_path():
     """Path to store zarr images written from benchmarks"""
     return pathlib.Path("data/output/heart-example.zarr")
+
+
+def expand_min_max(config: dict) -> dict:
+    """Expand min/max keys in config to a list of the full range of values."""
+
+    for key in ["chunk_size", "blosc_clevel", "gzip_level", "zstd_level"]:
+        values = config[key]
+        if "min" in values and "max" in values:
+            config[key] = range(values["min"], values["max"] + 1)
+
+    return config
+
+
+def parse_config_files(config_name: str) -> list[dict]:
+    """Parse json config files. 'all' will parse every config in tests/benchmark_configs (except for dev, which contains
+    a small set of parameters for development runs)."""
+
+    configs_dir = pathlib.Path("tests/benchmarks/benchmark_configs")
+    configs = []
+
+    if config_name == "all":
+        for config_file in configs_dir.iterdir():
```

Comment:
If instead you want a one-liner:
```
configs = [expand_min_max(read_json_file(config_file)) for config file in [f for f in configs_dir.glob("*.json") if f.stem != 'dev']]
```

but now I've written it it seems a bit silly 😄